python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -629,8 +629,8 @@ def __init__(self):
     @export
     @annotate_args([
         None,
-        ([-1, -1, -1], torch.float32, True),
-        ([-1, -1, -1], torch.int64, True),
+        ([2, 3, 4], torch.float32, True),
+        ([1, 2, 3], torch.int64, True),
```

Comment:
Please don't modify the existing tests. By making these static, the dynamic implementation in other backends is no longer being thoroughly tested. If you need a static test, you should add a new one

Reply:
Done

Reply:
Any idea to fix this dynamic shape e2e bug?
`python: /main_checkout/torch-mlir/build/tools/mlir/include/mlir/IR/BuiltinTypeInterfaces.h.inc:159: int64_t mlir::ShapedType::getNumElements() const: Assertion `hasStaticShape() && "cannot get element count of dynamic shaped type"' failed.`

Reply:
Resolved